force-app/main/default/classes/OSF_CaseController.cls

```diff
@@ -0,0 +1,50 @@
+/**********************************************************************
+ * @Name         : OSF_CaseController
+ * @Description  : Class containing callable functions from LWC.
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 01 jul. 2022
+ * @Modification Log: 
+ ********************************************************************
+ * Version		Developer		Date			Description
+ *___________________________________________________________________
+ *
+ *********************************************************************/
+public with sharing class OSF_CaseController {
+/**********************************************************************
+ * @Name         : getCases
+ * @Description  : Returns list of not closed cases 
+ *
+ * @Created By   : Marta Demeny
+ * @Created Date : 01 jul. 2022
+ *
+ * @param String - string containing customer id
+ * @return List<Case> - list of result objects
+ *********************************************************************/
+    @AuraEnabled(cacheable=true)
+    public static List<Case> getCases(String ownerId) {
+
+        return [
+            SELECT Id, CaseNumber, Origin, Status
+            FROM Case
+            WHERE Status != 'Closed'
+            AND Owner.id = :ownerId
+        ];
+    }
```

Comment:
OWner id is not the right field to query the correct cases as the service agent or queue gets the ownership not the creater. createdbyid can be a better fit as the component will create the case on the logged in user session

Reply:
done